### ISSUE TITLE
Reading styles/color information [feature request?]

### ISSUE THREAD

**Issue by rob2d** [catamphetamine/read-excel-file#1]
Hi,

Just wondering: is it possible with the library to read styles for cells? This is kind of critical for certain functionality with XLSX sheets (e.g. row background colors/highlighting to representing specific meaning in rows or cells), so if not, would be it be trivial or feasible to implement?

Thanks.

**Comment by catamphetamine** [comment]
Hi.
I myself don't know anything about the internals of XLSX files.
I just copy-pasted the parsing code and that was it.
The color can hold some meaning, that can be a valid use-case.
However, you'll have to find a person who knows how to parse a color of a cell if you want that.
One could tap into the `readXlsx()` function with `console.log(entries)` and perhaps figure something out from there:
https://github.com/catamphetamine/read-excel-file/blob/4cd58dd526f9f10da86a5e228cd61285f577c4df/source/readXlsx.js#L12

**Comment by rob2d** [comment]
Okay, thanks for the direction. I'll look into that.

**Comment by catamphetamine** [comment]
Some notes to whoever comes here.

So, seems that cell background colors are stored in ["cell styles"](https://stackoverflow.com/questions/6670144/microsoft-xml-excel-row-formatting) which can be obtained as `contents['xl/styles.xml']` inside `./readXlsx.js`.

https://stackoverflow.com/questions/19731176/xlsx-styles-getting-wrong-fill-pattern

There's a list of `<fills/>` which are bound to cells (`<c/>`) via `s` attribute (`s` being the id of the `<fill/>`)

![image](https://user-images.githubusercontent.com/477167/41101801-e348ecfc-6a6d-11e8-9e49-5d00b18a2ed3.png)

In the latest release `1.3.6` I added `verbose` option to `readXlsx()` which returns an object of shape `{ data, cells }` with `cells` holding the `backgroundColor` value.

Seems that there's a lot of ways of specifying a cell background color so I only added the simplest one of them as an illustration of the concept.

Overall, I wouldn't rely on parsing colors preferring explicitly set column values instead.
Still, seems that colors are somewhat standardised across the various software.
E.g. the conventional "yellow" being the acid `rgb(255, 255, 0)` one.
